### ISSUE TITLE
Client.boot fixture out of date, causing a test failure

### ISSUE THREAD

**Issue by Yhprum** [jackellenberger/emojme#68]
/spec/fixtures/clientBoot.json is a bit outdated - namely, the `prefs` object has been moved out of the `self` object and is now a key in the base object (this is what is causing the error right now). My own client.boot output seems to be 1168 lines compared to the 633 in the fixture, idk if you omitted anything though. 

Solutions could be to just move the offending `prefs` object to it's rightful place, or to just grab the new client.boot (though it would be a bit time consuming to sanitize all my personal info out of it tbh). What do you think?

**Comment by jackellenberger** [comment]
yeah, i think unfortunately the client boot fixture is a bit hand crafted at the moment. If memory service I pulled down a live one, then got tired sanitizing it 😬 I think making a minimal one that satisfies the tests (and of course updating the tests to expect real life content) would be a good move.
